게시판 만들기.py

```diff
@@ -0,0 +1,55 @@
+class Post:
+    def __init__(self, title, body):
+        self.title = title
+        self.body = body
+
+
+class Service:
+    def __init__(self):
+        self.post_list = []
+
+    def add_post(self, title, body):
+        if not (title and body):
+            print("[!] 제목과 본문은 공백일 수 없습니다!")
+
+        new_post = Post(title, body)
+        self.post_list.append(new_post)
+
+    def read_posts(self):
+        print("- 출력 결과 -")
+        self.get_posts()
+
+    def get_posts(self):
+        for ele in self.post_list:
+            index = self.post_list.index(ele)
+            print(f"{index:>3}번 포스트 \t 제목: {ele.title} / 본문: {ele.body}")
+
+    def delete_post(self, index):
+        target = self.post_list[index]
+        self.post_list.remove(target)
+
+    def exe_command(self):
+        while True:
+            print("\n어떤 명령을 실행하시겠습니까? \n(1.글 추가  2.글목록 조회  3.글 삭제  4.프로그램 종료)")
+            num = input("(원하는 명령의 숫자를 입력하세요) : ").strip()
+
+            if num == '1':
+                t, b = input("글의 제목을 입력하세요 : "), input("글의 본문을 입력하세요 : ")

```

Comment:
```suggestion
                t = input("글의 제목을 입력하세요 : ")
                b = input("글의 본문을 입력하세요 : ")
```

파이썬 문법을 잘 써먹은거라 생각 자체는 잘 했는데, 클린 코드 입장에서는 차라리 이렇게 두 줄로 나눠 쓰는 게 더 깔끔할듯.

Reply:
그리고 `t`, `b`는 어떤 단어를 축약한거? 변수 이름은 꼭 말이 통하게 네이밍해주세요!